pyproject.toml

```diff
@@ -21,8 +22,12 @@ extend_skip_glob = [
   "mlir/build/*",
   "runtime/build/*",
   ".git/*",
-  ".vscode/*"
+  ".vscode/*",
+  "*venv*/*",
+  "_deps/*",
 ]
+known_first_party = ["catalyst"]
+known_third_party = ["autoray", "jax", "pennylane"]
```

Comment:
Ah good, I imagine this fixed some confusion isort had about what package is third party?

Reply:
I would say `py_version` is most important, then this helps too.

Reply:
The presence of autoray is a bit strange though, it's not a package used by catalyst. The main dependencies are: jax, jaxlib, pennylane, diastatic-malt

Reply:
For some reason, my local isort thinks `autoray` (which is pulled by Catalyst) is a first party package. I would suggest adding third parties that confuse isort until we've converge on a stable config.

Reply:
But is it used anywhere in the catalyst, as in where does it actually sort it as a first party package? 

The pin was temporary and is being removed: #795

Reply:
Ah I know what happened, my bad. I'll accept your suggestion.